package.json

```diff
@@ -4,7 +4,7 @@
   "description": "An UI for cloud connector",
   "main": "index.js",
   "scripts": {
-    "test": "jest",
+    "test": "jest --no-cache",
```

Comment:
Just A side note. Can we add this flag in the job that is running triggering the test script? INstead of turning off the cache for everyone?

Reply:
Good point! Adding new script and `ci.sh` to run instead of default flow